xcode/Subconscious/Shared/Components/FirstRun/FirstRunView.swift

```diff
@@ -124,6 +166,9 @@ struct FirstRunView: View {
                     FirstRunDoneView(app: app)
                 }
             }
+            .onAppear {
+                app.send(.createSphere)
+            }
```

Comment:
I don't like this, there must be a better place?

I made this action idempotent in the `update` function so maybe we just call it at app startup?